src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -78,6 +78,8 @@
     )
 
     var (
+        Done = errors.New("iterator done")
```

Comment:
This was a guideline in the iterator design doc Jonathan sent out. There is a rationale for using a special sentinel value there.

I'm not personally opposed to using `io.EOF`, but the argument is that it doesn't make sense in an iterator context, and a special sentinel value doesn't really belong in cloud or gax.